script/rtspServer.js

```diff
@@ -0,0 +1,15 @@
+const Stream = require('node-rtsp-stream');
+
+// Get node args
+const args = process.argv.slice(2);
+
```

Comment:
Add the usage string
```js
// Check if required arguments are present
if (args.length < 2) {
  const filename = __filename.split('/').pop();
  console.error(`Usage: node ${filename} <rtsp-url> <websocket-port>`);
  process.exit(1);
}
```

Reply:
This makes it a bit more robust I'll add it!